tests/ZeroC.Slice.Tests/CustomTypeTests.cs

```diff
@@ -158,7 +158,7 @@ public void Decode_class_with_custom_type_fields()
         CustomTypeSliceEncoderExtensions.EncodeCustomType(ref encoder, myCustomType);
         CustomTypeSliceEncoderExtensions.EncodeNullableCustomType(ref encoder, null);
         structWithCustomTypeField.Encode(ref encoder);
-        encoder.EncodeTagged(1, TagFormat.FSize, myCustomType, CustomTypeSliceEncoderExtensions.EncodeCustomType);
+        encoder.EncodeTagged(1, TagFormat.FSize, myCustomType, (ref SliceEncoder encoder, MyCustomType value) => CustomTypeSliceEncoderExtensions.EncodeNullableCustomType(ref encoder, value));
```

Comment:
This line is too long

```suggestion
        encoder.EncodeTagged(
            1,
            TagFormat.FSize,
            myCustomType, 
            (ref SliceEncoder encoder, MyCustomType value) =>
                CustomTypeSliceEncoderExtensions.EncodeNullableCustomType(ref encoder, value));
```